### DISCUSSION TITLE
Does Aria Drag and Drop works with virtualised Table

### DISCUSSION THREAD

**Discussion by imjhonny** [adobe/react-spectrum#7545] [Q&A] [answered]
I am working on a virtualised Table using TansStacks Table and Virtualizer. However, I want to use the Aria Drag and Drop with the virtual rows. 
Could not find in the documentation but does Aria DnD works with Virtualization?

**Comment by snowystinger** [comment, marked as answer]
Yes, it should work. I'm not sure exactly how easily it will work as I don't believe anyone on the team has built a Table with TanStack's virtualizer.

Is there a particular issue you're running into? or do you have a codesandbox/stackblitz that you're looking for comments on?

We're always trying to improve our docs, that information should be coming in the future as we implement our own tableview's drag and drop in React Spectrum's latest iteration.

**Reply by imjhonny** [reply]
Yes the particular issue would be the displaying the DragPreview of an item that is virtualised. Since the item as you scroll will be unRendered, I can't keep the reference link between the useDrag and the DragPreview element.

**Reply by snowystinger** [reply]
can you use https://react-spectrum.adobe.com/react-aria/Table.html#custom-drag-preview ?
